stations.csv

```diff
@@ -30471,3 +30471,5 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 32618;Vila Cova de Lixa;vila-cova-de-lixa;;;41.3238028;-8.14665930000001;;PT;Europe/Lisbon;t;f;f;t;f;f;;;f;;f;;f;ez65nd;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 32619;Călan;calan;;;45.7403065;23.0017943;;RO;Europe/Bucharest;t;f;f;t;f;f;;;f;;f;;f;u80k5q;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 32620;Moscow;moscow;;;55.75222;37.61556;;RU;Europe/Moscow;t;f;f;t;f;f;;;f;;f;;f;ucftpv;t;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;Moskau;;;Moscou;Mosca;;Moskou;Moskva;Moskva;Moszkva;モスクワ;모스크바;Moskwa;;;Moskva;Moskova;莫斯科
+32621;Wasserburg Busbahnhof (Bodensee);wasserburg-busbahnhof-bodensee;;;47.570277;9.640427;7379;DE;Europe/Berlin;f;f;f;t;f;f;;;f;;f;990145;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
it doesn't have to be neither enabled nor suggestable since it's a bus station and the parent station is correctly handled by DB